setup.py

```diff
@@ -39,7 +39,7 @@ def read(fname):
     keywords=['socorro', 'breakpad', 'crash', 'reporting', 'minidump',
               'stacktrace'],
     packages=find_packages(),
-    install_requires=[], # use peep -r requirements.txt instead
+    install_requires=[], # use pip -r requirements.txt instead
```

Comment:
Well, I don't think we want people to be able to ``pip install socorro``, to be honest.